control/control.openSUSE.xml

```diff
@@ -192,6 +192,15 @@ textdomain="control"
                 <name>Source Repository</name>
                 <enabled config:type="boolean">false</enabled>
             </extra_url>
+            <extra_url>
+                <baseurl>https://codecs.opensuse.org/openh264/openSUSE_Leap</baseurl>
+                <alias>repo-openh264</alias>
+                <name>openSUSE-Leap-openh264</name>
```

Comment:
The name is allowed to contain spaces, it should contain the repository name in a human readable form. I think we should use a better name, see the other repositories. At least replace the dashes with spaces...